tests/test_data/python-utest_epel7.spec

```diff
@@ -0,0 +1,44 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name utest
+
+Name:           python-%{pypi_name}
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        Micro test module
+
+License:        GPLv2+
+URL:            https://github.com/fedora-python/pyp2rpm
+Source0:        %{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+
+%description
+
+
+%package -n     python%{python3_pkgversion}-%{pypi_name}
+Summary:        Micro test module
+
+Requires:       python%{python3_pkgversion}-pyp2rpm >= 3.3.1
+Conflicts:      python%{python3_pkgversion}-pyp2rpm >= 3.4
```

Comment:
Shouldn't this be: Requires: python%{python3_pkgversion}-pyp2rpm < 3.4

?

Reply:
Yeah, I selected the requires/conflicts pair based on my reading of Neal's link (July 27).  I offered to change that, but I think I was waiting for confirmation that was the preferred approach.  I've pushed a change for that.